extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -164,8 +164,9 @@ public function getArticleComments() {
 	private function prepareWikiVariables() {
 		$wikiVariables = $this->mercuryApi->getWikiVariables();
 		$navigation = $this->getNavigation();
-		if ( empty( $navData ) ) {
-			\Wikia\Logger\WikiaLogger::instance()->error(
+
+		if ( empty( $navigation ) ) {
+			\Wikia\Logger\WikiaLogger::instance()->warning(
```

Comment:
See https://wikia-inc.atlassian.net/wiki/display/PLAT/Logging#Logging-severitylevelsMeaningfulseveritylevels

IMO this should be `debug` level as there is no action we can take here.